cirq_superstaq/compiler_output.py

```diff
@@ -155,6 +155,11 @@ def _sequencer_from_state(state: Dict[str, Any]) -> "qtrl.sequencer.Sequence":
             state["_readout"] = readout_seq
 
         seq = _sequencer_from_state(state)
+    else:
+        warnings.warn(
+            "Your sequence for this output is None. Please make sure you have the qtrl package "
+            "installed."
```

Comment:
ah sorry one more nit
```suggestion
            "installed in order to de serialize compiled pulse sequences."
```